msgraph/accesspackageresourcerolescope.go

```diff
@@ -111,16 +111,16 @@ func (c *AccessPackageResourceRoleScopeClient) Create(ctx context.Context, acces
 // Get retrieves a AccessPackageResourceRoleScope.
 func (c *AccessPackageResourceRoleScopeClient) Get(ctx context.Context, accessPackageId string, id string) (*AccessPackageResourceRoleScope, int, error) {
 	resp, status, _, err := c.BaseClient.Get(ctx, GetHttpRequestInput{
+		OData: odata.Query{
+			Expand: odata.Expand{
+				Relationship: "accessPackageResourceRoleScopes",
+				Select:       []string{"accessPackageResourceRole", "accessPackageResourceScope"},
+			},
+			//Filter: fmt.Sprintf("startswith(originId,'%s')", id),
```

Comment:
do we need this?

Reply:
Probably not, I think it's a vestige of testing the API

```suggestion
```